DependencyInjection/Configuration.php

```diff
@@ -25,6 +25,9 @@ public function getConfigTreeBuilder()
                     ->defaultValue('365')
                     ->info('how many days banner should be hidden when user accepts policy?')
                 ->end()
+                ->integerNode('path')
```

Comment:
AFAIK this is a `scalarNode`

would be great if you could also provide some details using `info` key